src/lib/encrypt.py

```diff
@@ -0,0 +1,13 @@
+from cryptography.hazmat.primitives.ciphers import Cipher, algorithms, modes
+
+
+def symmetric_encrypt(key, iv, message):
+    """AES256 encrypt message and return ciphertext
+    params are all str
+    ciphertext returned is in bytes
+    """
```

Comment:
```suggestion
def symmetric_encrypt(key: str, iv: str, message: str) -> bytes:
    """AES256 encrypt a message.
    Returns
    -------
    ciphertext: bytes
    """
```

Reply:
Switches to using a typed funciton signature and use a standard docstring format ([numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) in this case, which is commonly used in python and IDEs support).